### DISCUSSION TITLE
Break in compatibility between 5.6.1 and 5.6.2?

### DISCUSSION THREAD

**Discussion by ves011** [mqttjs/MQTT.js#1875] [General]
My site stop working few days back for unknown reasons, after more than 1 year.
I'm using mosquitto broker and apache as websocket proxy.
After checks and checks i found if using 5.6.1 its  working fine, but with latest (5.6.2) it doesnt.
Attached  are some logs collected for failure case. 
[proxy.gnet-1716878789579.log](https://github.com/mqttjs/MQTT.js/files/15465222/proxy.gnet-1716878789579.log)

**Comment by robertsLando** [comment]
Could be due to: https://github.com/mqttjs/MQTT.js/pull/1874

**Comment by robertsLando** [comment]
Try with 5.7.0

**Comment by ves011** [comment]
replacing "https://unpkg.com/mqtt@5.6.1/dist/mqtt.min.js" with "https://unpkg.com/mqtt/dist/mqtt.min.js" (which should go to latest) it works!

THANKS!
